crates/jstzd/src/task/octez_client.rs

```diff
@@ -97,6 +101,61 @@ impl fmt::Display for Signature {
     }
 }
 
+#[derive(Debug)]
+pub struct Address {
+    pub hash: public_key_hash::PublicKeyHash,
+    pub public_key: public_key::PublicKey,
+    pub secret_key: Option<secret_key::SecretKey>,
+}
+
+impl Address {
+    const HASH: &'static str = "Hash";
```

Comment:
nit: maybe we should add a suffix to these constant names to clarify that these are the headers/names of the values printed, though it should also be fine without this since these constants are not public.